android/src/main/java/com/rudderstack/react/android/RNRudderSdkModule.java

```diff
@@ -144,14 +144,14 @@ public void identify(String userId, ReadableMap traits, ReadableMap options) {
     }
 
     @ReactMethod
-    public void getRudderContext(Callback callback) throws JSONException {
+    public void getRudderContext(Promise promise) throws JSONException {
```

Comment:
Please put the reason for using Promise instead of Callback. It'll be helpful to remember later on.

Reply:
Yeah, will add it